src/Microsoft.Management.Deployment/PackageCatalogReference.cpp

```diff
@@ -0,0 +1,82 @@
+#include "pch.h"
+#include <AppInstallerRepositorySource.h>
+#include "PackageCatalogReference.h"
+#include "PackageCatalogReference.g.cpp"
+#include "PackageCatalogInfo.h"
+#include "PackageCatalog.h"
+#include "ConnectResult.h"
+#include "Converters.h"
+#include "Microsoft/PredefinedInstalledSourceFactory.h"
+#include <wil\cppwinrt_wrl.h>
+
+namespace winrt::Microsoft::Management::Deployment::implementation
+{
+    void PackageCatalogReference::Initialize(
+        ::AppInstaller::Repository::SourceDetails sourceDetails,
+        winrt::Microsoft::Management::Deployment::PackageCatalogInfo packageCatalogInfo)
+    {
+        m_sourceDetails = sourceDetails;
+        m_info = packageCatalogInfo;
```

Comment:
The details and the info are the same data; I assume that this was just to prevent the need to convert an info back into a details?